ckanext/report/logic/action/get.py

```diff
@@ -15,6 +15,10 @@ def report_list(context=None, data_dict=None):
 
     registry = ReportRegistry.instance()
     reports = registry.get_reports()
+
+    user = context['auth_user_obj']
+    reports = filter(lambda r: r.is_visible_to_user(user), reports)
```

Comment:
Is this the right place to do the filtering or would it be better pushed back into get_reports ?